app/src/test/java/com/example/samplegallery/Utilities/LruBitmapCacheTest.java

```diff
@@ -0,0 +1,48 @@
+package com.example.samplegallery.Utilities;
+
+import android.content.Context;
+import android.icu.util.RangeValueIterator;
+import android.util.DisplayMetrics;
+
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+import java.lang.reflect.Method;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.*;
+
+/**
+ * Created by parth on 1/22/2017.
+ */
+
+public class LruBitmapCacheTest {
+
+    @Mock
+    Context mMockContext;
+
+    @Test
+    public void getCacheSize_isCorrect() throws Exception {
+
+        mMockContext = Mockito.mock(Context.class, Mockito.RETURNS_DEEP_STUBS);
+
+        DisplayMetrics dm = Mockito.mock(DisplayMetrics.class, Mockito.RETURNS_DEEP_STUBS);
```

Comment:
Why do you need deep stubs on DisplayMetrics? You're not really even calling any method of it!